VATSpy.dat

```diff
@@ -207,6 +207,7 @@ Papua New Guinea|AY|Control
 Paraguay|SG|
 Peru|SP|Control
 Philippines|RP|Control
+Phnom Penh|VD|

```

Comment:
We already have VD defined as:
Cambodia|VD|Control


Phnom Penh is not a country but a city.

Reply:
Will remove it due to dupelicate entry - submit a new PR if any change is needed from Colombia